### DISCUSSION TITLE
Additional URLs for autofill.

### DISCUSSION THREAD

**Discussion by Makishima** [keepassxreboot/keepassxc#9793] [Q&A] [answered]
Is it possible to configure secondary, tertiary, etc., URLs so that a browser extension can automatically fill in fields when authentication takes place on a domain other than the main site?

Similar to how Keepass2Android does.

**Comment by michaelk83** [comment, marked as answer]
https://keepassxc.org/docs/KeePassXC_UserGuide#_advanced_usage

**Reply by Makishima** [reply]
Thanks
